tests/blackbox/data/modbus/test_values/rpc/holding_registers_values_writing_big.json

```diff
@@ -6,7 +6,7 @@
   "16float": 0.067017,
   "32int": -856817645,
   "32uint": 2018915246,
-  "32float": -0.000163,
+  "32float": -0.020163,
```

Comment:
Changes in these files are not related to this functionality and they already in the master branch, you can try to pull changes from master and push to your branch.